pytezos/contract/metadata.py

```diff
@@ -0,0 +1,65 @@
+import json
+from os.path import dirname, join
+from typing import Any, Dict, List, Optional, Union
+
+from attr import dataclass
+from jsonschema import validate as jsonschema_validate  # type: ignore
+
+metadata_json_replace_table = {
+    '"return-type":': '"returnType":',
```

Comment:
I would not indulge using kebab case, the current consensus about all metadata (even beyond tzip-16) is that camel case should be used.

Reply:
This is required by jsonschema shipped with proposal.